tests/validation/tests/v3_api/test_workload.py

```diff
@@ -945,10 +945,7 @@ def test_rbac_wl_daemonset_list(role, remove_resource):
     user_token = rbac_get_user_token_by_role(role)
     p_client = get_project_client_for_token(project, user_token)
     remove_resource(workload)
-    if role == CLUSTER_MEMBER:
-        assert len(p_client.list_workload().data) == 1
-    else:
-        assert len(p_client.list_workload().data) == 2
+    assert len(p_client.list_workload(uuid=workload.uuid).data) == 1
```

Comment:
workload is removed even before it is listed. It should be the last step.

Reply:
Addressed in changes.